charts/stash/templates/deployment.yaml

```diff
@@ -19,9 +19,9 @@ spec:
     metadata:
       labels:
         {{- include "stash.labels" . | nindent 8 }}
-{{- if or .Values.annotations (and .Values.criticalAddon (eq .Release.Namespace "kube-system")) }}
+{{- if or .Values.annotations .Values.criticalAddon }}
```

Comment:
critical addons were restricted to kube-system namespace until 1.17 .
https://github.com/kubernetes/kubernetes/issues/60596#issuecomment-559537077

So, you need to check for k8s version before relaxing this. Otherwise pods will not start in < 1.17 release.

Reply:
Example: https://github.com/stashed/installer/blob/master/charts/stash/templates/mutating-webhook.yaml#L2-L3